extensions/interactions/LogicProof/static/js/conversion.js

```diff
@@ -21,30 +21,30 @@ var logicProofConversion = (function() {
   // NOTE: the 'old' values must all be single characters
   var REPLACEMENT_PAIRS = [{
     old: '&',
-    new: '\u2227'
+    "new": '\u2227'
```

Comment:
Sorry about this nit, but could you please use single quotes, per the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#general)? Thanks!

Everything else looks great :-)